functional/test

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash -e
+
+CDIR=$(cd `dirname $0` && pwd)
+USER_ID=${SUDO_UID:-$(id -u)}
+HOME=$(getent passwd "${USER_ID}" | cut -d: -f6)
+
+cd ${CDIR}/../
```

Comment:
If you're going to the trouble of changing to the root of the repo here, then just use relative paths for the rest of the script. Otherwise it's too much to maintain